packages/@aws-cdk/aws-codebuild/lib/build-spec.ts

```diff
@@ -1,4 +1,5 @@
 import { IResolveContext, Lazy, Stack } from '@aws-cdk/core';
+import {toYAML} from "../../../aws-cdk/lib/serialize";
```

Comment:
The `aws-cdk` is the CLI for CDK. You can't depend on it from a framework package like `@aws-cdk/aws-codebuild`.

Instead, depend on the package [that method uses](https://github.com/aws/aws-cdk/blob/623675d2f8fb2786f23beb87994e687e8a7c6612/packages/aws-cdk/lib/serialize.ts#L8-L8), that is, the `@aws-cdk/yaml-cfn` package.